courses/machine_learning/deepdive2/structured/solutions/3_automl_tables_babyweight.ipynb

```diff
@@ -0,0 +1,487 @@
+{
```

Comment:
<p>Maybe I'm part of an A/B test, but my UI was different (didn't have SCHEMA). Could you do me a favor and verify that the UI is still the same for you?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I took that screenshot just two days ago, however I did it through the Public Cloud Console (I used a qwiklabs account to make all of my labs so that it would be the TRUE user experience). Maybe you are using the Corp Cloud Console and seeing a different UI? I just checked again with the qwiklabs account and it is still exactly the same as my screenshot.</p>

Reply:
Yeah, that's probably it. Probably best to stick with the public version :)